
{'date': 'Tue Nov 20 2018 12:12:50.710 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 348
    circus(0)
  module <module> line 344
    DES[desafio](param)
  module <module> line 329
    Masmorra()
  module <module> line 22
    self.gamer = Braser(800, 600)
TypeError: 'module' object is not callable
'''},
{'date': 'Tue Nov 20 2018 12:28:22.714 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 348
    circus(0)
  module <module> line 344
    DES[desafio](param)
  module <module> line 329
    Masmorra()
  module <module> line 24
    self.hero = Hero(self)
  module <module> line 158
    self.gamer = gamer.gamer
AttributeError: 'Masmorra' object has no attribute 'gamer'
'''},
{'date': 'Tue Nov 20 2018 12:34:32.902 GMt-0200 (Horário de Verão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 352
    circus(0)
  module <module> line 348
    DES[desafio](param)
  module <module> line 333
    Masmorra()
  module <module> line 26
    self.gamer = Braser(800, 600, [self.Hero, self.Monster])
AttributeError: 'Masmorra' object has no attribute 'Hero'
'''},
{'date': 'Thu May 23 2019 10:38:41.96 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    peixe1 = Elemento(BICHO, x=100, y=100, w=60, h=40)
NameError: name 'BICHO' is not defined
'''},
{'date': 'Thu May 23 2019 10:46:00.457 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    jogoPescaria()
  module <module> line 11
    pescaria = Cena(CENA_PESCARIA, x=500, y=500, w=250, h=200)
  module _spy.vitollino.main line 778
    Cena.c(**kwargs)
  module _spy.vitollino.main line 821
    imagem, kwargs = (imagem, {}) if isinstance(imagem, str) \
AttributeError: 'int' object has no attribute '__getitem__'
'''},
{'date': 'Thu May 23 2019 10:59:13.954 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  def __init__(self, pescaria)
                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu May 23 2019 11:11:33.479 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 34
    jogoPescaria()
  module <module> line 31
    Crianca(crianca)
NameError: name 'crianca' is not defined
'''},
{'date': 'Thu May 23 2019 11:12:06.786 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 34
    jogoPescaria()
  module <module> line 31
    Crianca(crianca)
NameError: name 'crianca' is not defined
'''},
{'date': 'Thu May 23 2019 11:15:42.215 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 16
  def __init__(self, pescaria,x=0, y=230)
                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu May 23 2019 11:16:04.876 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 34
    jogoPescaria()
  module <module> line 31
    Crianca(crianca)
NameError: name 'crianca' is not defined
'''},